avalon/tools/cbsceneinventory/app.py

```diff
@@ -112,7 +164,7 @@ def show_right_mouse_menu(self, pos):
 
         # move index under mouse
         indices = self.get_indices()
-        if not active.parent().isValid():
+        if active not in indices:
```

Comment:
This change still "invalidates" whatever it tries to do in the lines below. I think this might be messing with an edge case that was previously there? The lines below the assertion will never run, because it will always hit the assert in this is `if` statement?

Reply:
* Assume active is valid? If not, don't show a menu?
* if active in selection: place active to the end
* active not in selection: we don't care

The docstrings states that the menu should only be displayed when the position of the item is clicked so I think this should fix it

Reply:
> active not in selection: we don't care

Or we place it add the end